src/login/logind-button.c

```diff
@@ -14,7 +14,7 @@
 #include "string-util.h"
 #include "util.h"
 
-#define CONST_MAX4(a, b, c, d) CONST_MAX(CONST_MAX(a, b), CONST_MAX(c, d))
+#define CONST_MAX5(a, b, c, d, e) CONST_MAX(CONST_MAX(CONST_MAX(a, b), CONST_MAX(c, d)), e)
```

Comment:
I think it would be much cleaner, error-prone and the like to keep `CONST_MAX4` as-is and use this here:
`#define CONST_MAX5(a, b, c, d, e) CONST_MAX(e, CONST_MAX4(a, b, c, d))`